### DISCUSSION TITLE
Duplicated Events

### DISCUSSION THREAD

**Discussion by derkrasseleo** [bitfireAT/icsx5#386] [Q&A] [answered]
The app has been working fine for me until I noticed that all events synced via icsx5 are duplicated now. I tried the Fossify Calendar and Etar, in both apps I see duplicated entries for each event. Is this a known issue? One calendar is ical and the other one is a Google calendar (ics). FYI: Events from the DAVX5 app are not duplicated. And of course I checked externally if there are duplicated events in the calendar.

**Comment by sunkup** [comment, marked as answer]
We have not heard of this yet, no. When did you notice this? Which ICSx5 app version are you on? 

You could try deleting the subscription (unsubscribing from the calendar) and adding it back. Since the events are all stored on the server. To copy the calendar address before unsubscribing, tap on the "share" icon next to the bin icon. You can then re-add the subscription by pasting the address in the "Webcal address" field.

**Reply by derkrasseleo** [reply]
Removing and readding fixed it for now. I actually noticed it about a week ago but thought it was a problem with the calendar itself. Version is 2.2.3-standard
